proto/tm2/multisig.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package tm;
+
+option go_package = "github.com/gnolang/gno/tm2/pkg/crypto/multisig/pb";
+
+// imports
+import "google/protobuf/any.proto";
+
+// messages
+message PubKeyMultisig {
+	uint64 k = 1 [json_name = "threshold"];
+	repeated google.protobuf.Any pub_keys = 2 [json_name = "pubkeys"];
+}
+
+message Multisignature {
+	CompactBitArray bit_array = 1;
+	repeated bytes sigs = 2;
+}
+
+message CompactBitArray {
```

Comment:
Just curious, where is this from?

Reply:
my hands

Reply:
I was worried about using uint32 as uint8 but turns out it's encoded the same by protobuf
The hex values from the test come from marshaling the type in go